mifospay/src/main/java/org/mifos/mobilewallet/mifospay/common/presenter/MakeTransferPresenter.kt

```diff
@@ -27,7 +27,7 @@ class MakeTransferPresenter @Inject constructor(private val mUsecaseHandler: Use
     }
 
     override fun fetchClient(externalId: String?) {
-        mUsecaseHandler.execute(searchClient, SearchClient.RequestValues(externalId),
+        mUsecaseHandler.execute(searchClient, externalId?.let { SearchClient.RequestValues(it) },
```

Comment:
I don't think it should be nullable if It should be then please handle gracefully pass null if it is null